tensorflow_tts/configs/fastspeech.py

```diff
@@ -16,7 +16,7 @@
 
 import collections
 
-from tensorflow_tts.processor.ljspeech import symbols
+from tensorflow_tts.processor.kss import symbols
```

Comment:
can you do the same as this PR (https://github.com/TensorSpeech/TensorFlowTTS/blob/dev/chinese_example/tensorflow_tts/configs/fastspeech.py).  ?